Dockerfile

```
@@ -0,0 +1,88 @@
+# syntax=docker/dockerfile:1
+
+ARG CUDA_VER=11.8.0
+ARG PYTHON_VER=3.10
+ARG LINUX_VER=ubuntu22.04
+
+ARG RAPIDS_VER=23.08
+ARG DASK_SQL_VER=2023.6.0
+
+ARG BASE_FROM_IMAGE=rapidsai/mambaforge-cuda
+
+# Gather dependency information
+FROM rapidsai/ci:latest AS dependencies
+ARG CUDA_VER
+ARG PYTHON_VER
+
+ARG RAPIDS_VER
+
+ARG RAPIDS_BRANCH="branch-${RAPIDS_VER}"
+
+RUN pip install --upgrade conda-merge rapids-dependency-file-generator
+
+COPY notebooks.sh /notebooks.sh
+
+RUN /notebooks.sh
+
+
+# Base image
+FROM ${BASE_FROM_IMAGE}:cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER} as base
+ARG CUDA_VER
+ARG PYTHON_VER
+
+ARG RAPIDS_VER
+ARG DASK_SQL_VER
+
+RUN useradd -rm -d /home/rapids -s /bin/bash -g conda -u 1001 rapids
```

```suggestion
RUN useradd -rm -d /home/rapids -s /bin/bash -g conda -u 1000 rapids
```

Is there anything preventing us from using `1000` here?

This would match the `id` used for the first user in OSes like Ubuntu.

---

Nope, `1000` is fine. I probably just copied-pasted this from somewhere.